### PR TITLE
Add note about required settings

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

**Comment by lubosmj** [comment]
The file docs/_static/api.json was changed in a very suspicious way. I built the docs locally and everything was updated as expected, though.

File: docs/installation.rst [hunk 1]
```diff
@@ -55,6 +55,12 @@ Make and Run Migrations
    django-admin makemigrations container
    django-admin migrate container
 
+Set up Required Settings
```
Comment (ipanova) [inline docs/installation.rst]: s/setup/ configure/

File: docs/installation.rst [hunk 2]
```diff
@@ -55,6 +55,12 @@ Make and Run Migrations
...
+------------------------
+
+The plugin expects to have defined additional settings. These settings are obligatory if a user wants
```
Comment (ipanova) [inline docs/installation.rst]: s/obligatory/required/

File: docs/installation.rst [hunk 3]
```diff
@@ -55,6 +55,12 @@ Make and Run Migrations
...
+to utilize the token authentication while managing content, see :ref:`authentication`.
```
Comment (ipanova) [inline docs/installation.rst]: s/to utilize/ to use/
s/managing/ serving/

File: docs/_static/api.json [hunk 4]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"swagger": "2.0", "info": {"title": "Pulp 3 API", "logo": {"url": "https://pulp.plan.io/attachments/download/517478/pulp_logo_word_rectangle.svg"}, "version": "v3"}, "host": "localhost:24817", "schemes": ["http"], "basePath": "/", "consumes": ["application/json"], "produces": ["application/json"], "securityDefinitions": {"Basic": {"type": "basic"}}, "security": [{"Basic": []}], "paths": {}, "definitions": {}, "tags": []}
```
Comment (ipanova) [inline docs/_static/api.json]: this does not look right, most of the stuff is removed for some reason, try to regenerate.

**Pull request merged**